README.rst

```diff
@@ -8,32 +8,35 @@ against different databases and ORMs.
 Installation and Use
 --------------------
 
-1. Make a Python 3.7 venv.
+1. Make a Python 3.8 venv (to be compatible with ``synth``).
 
 2. Install dependencies from ``requirements.txt``.
 
-3. Install EdgeDB, MongoDB, PostgreSQL 11, Golang toolchain.
+3. Install EdgeDB, MongoDB, PostgreSQL 13, Golang toolchain.
 
-4. Install NodeJS (at least v10.15.3) and TypeScript toolchains.
+4. Install NodeJS (at least v14.16.0) and TypeScript toolchains.
 
 5. Install docker and docker-compose (needed for Hasura and Prisma benchmarks).
 
-6. Install Prisma 1.26.4 (later versions don't work with timestamptz) via
-   ``npm i -g prisma@1.26.4``.
+6. Install Prisma (later versions don't work with timestamptz) via
```

Comment:
The comment in parens is probably outdated, no?

Reply:
Yeah, I removed the comment.